doc/nrf/releases_and_maturity/releases/release-notes-1.6.0.rst

```diff
@@ -158,7 +158,7 @@ nRF9160
     * Added polling "226 Transfer complete" after data channel TX/RX, with a configurable timeout of 60 seconds.
     * Ignored the reply code of "UTF8 ON" command as some FTP servers return abnormal replies.
 
-  * :ref:`at_params_readme` library:
+  * AT params library:
```

Comment:
Not deleted or is it? If not, leave as is.

Reply:
It is deleted

Reply:
AT parameters.

Reply:
Ok